src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEqualityContractProperty.cs

```diff
@@ -142,6 +137,7 @@ internal sealed class GetAccessorSymbol : SourcePropertyAccessorSymbol
                        usesInit: false,
                        isAutoPropertyAccessor: true,
                        isExplicitInterfaceImplementation: false,
+                       isNullableEnabled: false,
```

Comment:
Why would nullable be disabled here? Shouldn't it depend on the context the record was declared in? For example, if my project is nullable enabled I'd expect that this property returns `Type!`, not `Type~`/

Reply:
`isNullableEnabled: false` prevents running nullable analysis but it does not affect the member signature.

---
In reply to: [546876850](https://github.com/dotnet/roslyn/pull/49953#discussion_r546876850) [](ancestors = 546876850)